src/js/components/deployments/deployments.js

```diff
@@ -276,41 +301,28 @@ export default class Deployments extends React.Component {
     this.setState(this._getInitialState());
   }
 
-  _getGroupDevices(group) {
-    // get list of devices for each group and save them to state
-    var self = this;
-    return AppActions.getAllDevicesInGroup(group).then(devices => {
-      let state = {};
-      state[group] = devices;
-      self.setState(state);
-    });
-  }
-
-  dialogDismiss() {
-    this.setState({
-      reportDialog: false,
-      artifact: null,
-      release: null,
-      releaseArtifacts: null,
-      group: null
-    });
-  }
-
   _retryDeployment(deployment, devices) {
```

Comment:
when I merged the current master into this, the `_retryDeployment` changes were overwritten and I'm not sure why - just to keep it in mind...

Reply:
I'm not sure why it would happen, I rebased on latest master first

Reply:
then probably I did something wrong during the merge 🙈